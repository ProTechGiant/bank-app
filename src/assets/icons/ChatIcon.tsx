import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function ChatIcon({ color = "#1E1A25", height = 24, width = 25 }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M1.65002 16.3585V3.38352C1.65002 3.06685 1.76269 2.79585 1.98802 2.57052C2.21269 2.34585 2.48336 2.23352 2.80002 2.23352H15.925C16.2417 2.23352 16.5127 2.34585 16.738 2.57052C16.9627 2.79585 17.075 3.06685 17.075 3.38352V12.5085C17.075 12.8252 16.9627 13.0919 16.738 13.3085C16.5127 13.5252 16.2417 13.6335 15.925 13.6335H5.75002L2.62502 16.7585C2.44169 16.9585 2.23336 17.0085 2.00002 16.9085C1.76669 16.8085 1.65002 16.6252 1.65002 16.3585ZM7.02502 18.9085C6.70836 18.9085 6.44169 18.8002 6.22502 18.5835C6.00836 18.3669 5.90002 18.1002 5.90002 17.7835V15.6335H19.075V6.48352H21.2C21.5167 6.48352 21.7877 6.59185 22.013 6.80852C22.2377 7.02519 22.35 7.29185 22.35 7.60852V21.5835C22.35 21.8502 22.2334 22.0335 22 22.1335C21.7667 22.2335 21.5584 22.1919 21.375 22.0085L18.275 18.9085H7.02502ZM14.8 4.50852H3.92502V11.3585H14.8V4.50852Z"
        fill={color}
      />
    </Svg>
  );
}
