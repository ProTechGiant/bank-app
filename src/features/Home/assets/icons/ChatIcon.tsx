import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export default function ChatIcon({ width = 16, height = 16, color = "#FAFAFA" }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 16 16" fill={color} xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M13.334 1.33301H2.66732C1.93398 1.33301 1.33398 1.93301 1.33398 2.66634V14.6663L4.00065 11.9997H13.334C14.0673 11.9997 14.6673 11.3997 14.6673 10.6663V2.66634C14.6673 1.93301 14.0673 1.33301 13.334 1.33301ZM13.334 10.6663H4.00065L2.66732 11.9997V2.66634H13.334V10.6663Z"
        fill={color}
      />
    </Svg>
  );
}
