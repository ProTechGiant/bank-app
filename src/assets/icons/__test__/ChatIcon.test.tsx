import { create } from "react-test-renderer";

import { ChatIcon } from "../ChatIcon";

describe("icon/ChatIcon", () => {
  it("renders a ChatIcon", () => {
    const result = create(<ChatIcon />);

    expect(result).toMatchSnapshot();
  });
});
