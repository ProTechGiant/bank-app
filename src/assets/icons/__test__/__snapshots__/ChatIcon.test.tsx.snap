// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/ChatIcon renders a ChatIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="25"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 25,
      },
    ]
  }
  vbHeight={24}
  vbWidth={25}
  width={25}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M15.5 4V11H5.67L4.5 12.17V4H15.5ZM16.5 2H3.5C2.95 2 2.5 2.45 2.5 3V17L6.5 13H16.5C17.05 13 17.5 12.55 17.5 12V3C17.5 2.45 17.05 2 16.5 2ZM21.5 6H19.5V15H6.5V17C6.5 17.55 6.95 18 7.5 18H18.5L22.5 22V7C22.5 6.45 22.05 6 21.5 6Z"
      fill={
        {
          "payload": 4280823686,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
